sdk/services/futures.ts

```diff
@@ -843,4 +843,16 @@ export default class FuturesService {
 			amount.toBN(),
 		]);
 	}
+
+	// This is on an interval of 15 seconds.
+	public async getFuturesTrades(marketKey: FuturesMarketKey, minTs: number, maxTs: number) {
+		const response = await queryFuturesTrades(this.sdk, marketKey, minTs, maxTs);
+		return response ? mapTrades(response) : null;
+	}
+
+	public async getOrderFee(marketAddress: string, size: Wei) {
```

Comment:
This file has read functions at the top and write functions at the bottom, could we bump this up above the write calls?

Reply:
Sure. Pushed a commit that fixes this.